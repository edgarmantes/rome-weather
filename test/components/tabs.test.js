import React from 'react'
import renderer from 'react-test-renderer';
import Tabs from '../../src/components/tabs';

describe('<Tabs />' , () => {


	let wrapper

	beforeEach( ()=>{
		wrapper = shallow(<Tabs />)
	})

	it('Renders Tabs component properly', () => {
		expect(wrapper.length).toEqual(1)
	});

	// it('Adds Tabs passed in value to this.props.day', () => {
	// 	expect(wrapper.props().children).toEqual("list")
	// })

	it('Snapshot of Tabs', () => {
        const renderedValue =  renderer.create(<Tabs />).toJSON()
        expect(renderedValue).toMatchSnapshot();
    });
});
