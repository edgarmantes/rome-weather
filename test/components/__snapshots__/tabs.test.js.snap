// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tabs /> Snapshot of Tabs 1`] = `
<div
  className="tabs-container"
>
  <ul
    className="tabs"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <li
        className="tab-forecast tab"
      >
        Forecast
      </li>
    </a>
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <li
        className="tab-Precip tab"
      >
        Precipication
      </li>
    </a>
  </ul>
</div>
`;
